README.md

```diff
@@ -216,6 +216,36 @@ ENV BROKER_CLIENT_URL   http://my.broker.client:8000
 ENV PORT                8000
 ```
 
+### Artifactory 
+
+To use the the broker client with an artifactory deployment, run `docker pull snyk/broker:artifactory` tag. The following environment variables are needed to customize the broker client:
```

Comment:
the the the the the the the

Reply:
It's because it's very specific broker-client 😄